lizard_ext/lizardnd.py

```diff
@@ -30,7 +30,7 @@ def __call__(self, tokens, reader, l_depth=0):  # pylint: disable=R0912
         if hasattr(reader, "loops"):
             loops = reader.loops
         else:
-            loops = set(['if', 'else', 'foreach', 'for', 'while', '&&', '||',
+            loops = set(['if', 'foreach', 'for', 'while', '&&', '||',
```

Comment:
Why 'else' needs to be removed?

Reply:
I reasoned `else` did not change the nesting level. 

If I put `else` back the code `int fun(){if(a)xx;else b;}` shows a max level of 2 but I believe it should be 1, same as `int fun(){if(a) xx;}`. The relevant test case is `test_one_function_nd_with_else`.